package.json

```diff
@@ -21,5 +21,8 @@
     "husky": "^9.0.11",
     "lerna": "^6.6.2",
     "typescript": "^4.9.3"
+  },
+  "dependencies": {
+    "micromatch": "^4.0.8"
```

Comment:
Sorry to keep going back and forth on this, but could you remove this dependency?

Reply:
Absolutely, it was also a Dependency review failure, it must have been associated with `braces`